### DISCUSSION TITLE
Table Field, max columns?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#11204] [General]
Hi

Any chance we can add a max columns option in the Table Field? Other field types provide similar options.

**Comment by edalzell** [comment]
Sure, that's a reasonable request, add it here if there isn't one already: https://github.com/statamic/ideas/issues/new/choose, or try to PR it?

**Reply by stephenmeehanuk** [reply]
Done: https://github.com/statamic/ideas/issues/1276
